src/v/cloud_storage/segment_chunk_api.h

```diff
@@ -102,6 +97,21 @@ class segment_chunks {
     /// Hydrate a single chunk during an iteration of the bg loop. Uses remote
     /// segment to download the chunk and once finished notifies all waiters.
     ss::future<> do_hydrate_chunk(chunk_start_offset_t start_offset);
+
+    /// Schedules prefetches when a chunk is downloaded, by calculating the next
+    /// `prefetch` chunks and scheduling downloads by adding to the wait queue.
+    /// If prefetch chunks are already hydrated or download is in progress, then
+    /// we skip the download.
+    void
+    schedule_prefetches(chunk_start_offset_t start_offset, size_t prefetch);
```

Comment:
nit: may be clearer renamed `num_prefetches` or somesuch?

Reply:
changed